RELEASE_NOTES.md

```diff
@@ -44,6 +44,7 @@ This serves two purposes:
 ### Removed
 - Breaking: Removed the build task `\Hyde\Framework\Actions\PostBuildTasks\GenerateSearch` (see upgrade guide below)
 - Breaking: Removed the deprecated `\Hyde\Framework\Services\BuildService::transferMediaAssets()` method (see upgrade guide below)
+- Internal: Removed the deprecated global`unslash()` function, replaced with the namespaced `\Hyde\unslash()` function in https://github.com/hydephp/develop/pull/1754
```

Comment:
Should not be marked as internal, fixed in https://github.com/hydephp/develop/pull/1565/commits/ecbf74928f8714eec55cd9730f9a293629edb7aa